src/libs/Pusher/library/types.ts

```diff
@@ -0,0 +1,3 @@
+import Pusher from 'pusher-js/with-encryption';
```

Comment:
```suggestion
import PusherClass from 'pusher-js/types/src/core/pusher';

type Pusher = typeof PusherClass;
```

This way will allow you to correct import Pusher in `pusher.ts` and use better typing in the platform variant files.